README.md

```diff
@@ -1,75 +1,63 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Project: POrtfolio Project
 
-> Description the project.
+> This is my first webdesign and it happens to be my portforlio, _Though the skills are still in the oven_   :wink: .
+Hope you enjoy it. 
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+- HTML for content
+- CSS for styling
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
+What do you need ?
 
 ### Prerequisites
 
+No major requisites, an IDE like VSCode and a browser will do.
+
 ### Setup
+ 
+ Its plug an play
 
 ### Install
 
+As at now, no packages to install
+
 ### Usage
 
+Current, this is the mobile view 
+
 ### Run tests
 
-### Deployment
+Coming soon
 
+### Deployment
 
+Open `index.html` using a web browser to deploy the web page
 
 ## Authors
 
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-## 🤝 Contributing
+👤 **My Contacts**
 
-Contributions, issues, and feature requests are welcome!
+- GitHub: [@githubhandle](https://github.com/mfadhili)
+- Twitter: [@twitterhandle](https://twitter.com/Mfadhili3)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mfadhili-gitau-b9294122b/)
 
-Feel free to check the [issues page](../../issues/).
 
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- Updating this ...
 
 ## 📝 License
 
 This project is [MIT](./LICENSE) licensed.
 
```

Comment:
Kindly add an MIT license file with the name LICENSE file to the root directory of your project in order not to have an inactive link.